coremltools/converters/mil/frontend/tensorflow/ops.py

```diff
@@ -729,9 +729,16 @@ def DepthwiseConv2dNative(context, node):
 
     pad_type = pad_type.lower()
     x = context[node.inputs[0]]
-    C_in = x.shape[-1]
+
     if data_format == "NHWC":
         x = _transpose_NHWC_to_NCHW(x)
+        C_in = x.shape[-1]
+    elif data_format == "NCHW":
+        C_in = x.shape[1]
```

Comment:
I don't know what's possible, but if `data_format` is neither of `"NHWC"` or `"NCHW"`, then `C_in` will be an unbound variable